sdk/eventhub/azure-eventhub/azure/eventhub/_producer.py

```diff
@@ -264,7 +264,7 @@ def send(
                     links = [link] if link else []
 
             self._unsent_events = [wrapper_event_data._message]  # pylint: disable=protected-access
-            with send_context_manager(self._client, links=links):  # pylint: disable=protected-access
+            with send_context_manager(self._client, links=links):
```

Comment:
with the disable=protected-access gone here, can the one above also be removed?

Reply:
no